lib/api.js

```diff
@@ -21,7 +21,10 @@ export async function sign_up({ email, password, role }) {
 }
 
 export async function login({ email, password }) {
```

Comment:
```suggestion
export default async function login({ email, password }) {
```

Don't you need this?